run-bungeecord.sh

```diff
@@ -61,14 +61,15 @@ case "${TYPE^^}" in
   ;;
 esac
 
-if [ -f "$BUNGEE_JAR" ]; then
-  zarg="-z '$BUNGEE_JAR'"
-fi
```

Comment:
Removing this zarg logic breaks a feature that users requested, which is to ensure that the latest bungee jar is checked even if the file is already present. I can push some follow up changes, if want to leave this fix to me.